specs/rollup-node.md

```diff
@@ -92,40 +94,49 @@ Note that the sequencing windows overlap.
 
 The rollup reads the following data from the [sequencing window][g-sequencing-window]:
 
-- Of the *first* block in the window only:
-  - L1 block attributes:
-    - block number
-    - timestamp
-    - basefee
-    - *random* (the output of the [`RANDOM` opcode][random])
-  - L1 log entries emitted for [user deposits][g-deposits], derived transactions are augmented with
-    `blockHeight` and `transactionIndex` of the transaction in L2.
-- Of each block in the window:
-  - Sequencer batches, derived from the transactions:
-    - The transaction receiver is the sequencer inbox address
-    - The transaction must be signed by a recognized sequencer account
-    - The calldata may contain a bundle of batches. *(calldata will be substituted with blob data in the future.)*
-    - Batches not matching filter criteria are ignored:
-      - `batch.epoch == sequencing_window.epoch`, i.e. for this sequencing window
-      - `(batch.timestamp - genesis_l2_timestamp) % block_time == 0`, i.e. timestamp is aligned
-      - `min_l2_timestamp <= batch.timestamp < max_l2_timestamp`, i.e. timestamp is within range
-        - `min_l2_timestamp = prev_l2_timestamp + l2_block_time`
-          - `prev_l2_timestamp` is the timestamp of the previous L2 block: the last block of the previous epoch,
-              or the L2 genesis block timestamp if there is no previous epoch.
-          - `l2_block_time` is a configurable parameter of the time between L2 blocks
-        - `max_l2_timestamp = max(l1_timestamp + max_sequencer_drift, min_l2_timestamp + l2_block_time)`
-          - `l1_timestamp` is the timestamp of the L1 block associated with the L2 block's epoch
-          - `max_sequencer_drift` is the most a sequencer is allowed to get ahead of L1
-      - The batch is the first batch with `batch.timestamp` in this sequencing window,
-        i.e. one batch per L2 block number.
-      - The batch only contains sequenced transactions, i.e. it must NOT contain any Deposit-type transactions.
+From the *first* block in the window only:
+
+- L1 block attributes:
+  - `blockNumber`
+  - `timestamp`
+  - `basefee`
+  - `random` (the output of the [`RANDOM` opcode][random])
```

Comment:
The reason these were not in code case was that in my mind code case refers to "defined" terms.

They could be terms defined in the spec to be referred to later, or definitions in particular structures / code files.

It's not quite clear to me that this refers to a specific definition? If that's the case (e.g. the geth block structure) I think this is fine but we should link/mention the source of these definition.